src/pages/messages/components/PublishingRequestMessagesColumn.tsx

```diff
@@ -12,41 +13,48 @@ export const StyledMessagesContainer = styled(Box)({
 export const StyledStatusMessageBox = styled(Box)({
   display: 'grid',
   gridTemplateColumns: '1fr auto',
-  gap: '0.5rem 1rem',
+  gap: '0.25rem 0.5rem',
   padding: '0.2rem 1rem',
   borderRadius: '4px',
 });
 
 interface PublishingRequestMessagesColumnProps {
   ticket: ExpandedPublishingTicket | PublishingTicket;
+  showLastMessage?: boolean;
 }
 
-export const PublishingRequestMessagesColumn = ({ ticket }: PublishingRequestMessagesColumnProps) => {
+export const PublishingRequestMessagesColumn = ({ ticket, showLastMessage }: PublishingRequestMessagesColumnProps) => {
   const { t } = useTranslation();
 
   return (
     <StyledMessagesContainer>
       {ticket.status === 'Pending' || ticket.status === 'New' ? (
-        <StyledStatusMessageBox sx={{ bgcolor: 'secondary.dark' }}>
-          <Typography>
-            {ticket.filesForApproval.length
-              ? t('registration.files_and_license.files_awaits_approval', { count: ticket.filesForApproval.length })
-              : t('registration.files_and_license.files_awaits_approval_unknown')}
-          </Typography>
-        </StyledStatusMessageBox>
-      ) : (
-        <StyledStatusMessageBox sx={{ bgcolor: 'publishingRequest.main' }}>
-          {ticket.status === 'Completed' ? (
+        <>
+          {showLastMessage && <LastMessageBox ticket={ticket as ExpandedPublishingTicket} />}
+          <StyledStatusMessageBox sx={{ bgcolor: 'secondary.dark' }}>
```

Comment:
Her kan det være at man har lagt inn kommentar _etter_ at statusen ble satt, og sånn sett ville forventa at kommentaren kom under "2 filer venter verifisering" etc. Men ser at det ville komplisert veldig mye å få til det på en god måte, så bruker heller en universalløsning for alle tickets, med at melding alltid vises over status-beskrivelsen